### ISSUE TITLE
GKE integration periodic cleanup: IP Address + DNS

### ISSUE THREAD

**Issue by Tomasz-Smelcerz-SAP** [kyma-project/test-infra#275]
Ensure IP Address and DNS allocated by GKE integration jobs for testing Kyma are removed.
Configure a periodic job for the task.

AC:
- gather a list of IPs/DNS entries to remove
- remove DNS entry and IP from the list one by one

Note: An idea of how to find objects to remove: combine pieces of information from name pattern, and creation time. IP/DNS entry object name is the same as cluster name. You can use this information: If there's no cluster with such a name it means this IP/DNS entry pair is orphaned and can be removed.

**Comment by piotrmsc** [comment]
I would first check how the community do it, perhaps there's already a tool for that?
There are more k8s related products working with prow, I assume it's a more common problem.
If it's not there, I'm fine with doing code implementation but maybe as a contribution to prow etc.?

**Comment by Demonsthere** [comment]
I have asked the people over at istio what is their predisposition towards this problem: 
https://github.com/istio/test-infra/issues/1130
Regarding an existing cleaner, there is https://github.com/pantheon-systems/kube-gce-cleanup which cleans LoadBalancers exactly as the go program I have created. However, I have not found any existing cleaner for DNS entries

**Comment by Demonsthere** [comment]
We are putting this on hold for the time being, as https://github.com/kyma-project/kyma/issues/1871 may remove the need of using GCP DNS at all. If that element is added and we won't need the DNS, we won't need to remove any DNS related orphans, as none will be created.

**Comment by Tomasz-Smelcerz-SAP** [comment]
This issue is blocked by #362

**Comment by piotrmsc** [comment]
AC:

- review the code in the pr - requires DNS logic implementation , IP cleaner already works, write tests
- plug it to the prow
